src/reset.css

```diff
@@ -0,0 +1,4 @@
+body,
+div {
```

Comment:
div doesn't have default margins
```suggestion
body {
```